src/Microsoft.OData.Core/MultipartMixed/ODataMultipartMixedBatchReader.cs

```diff
@@ -0,0 +1,466 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ODataMultipartMixedBatchReader.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.MultipartMixed

```

Comment:
>Microsoft.OData.MultipartMixed [](start = 10, length = 30)

three classes doesn't seem sufficient for a namespace.

Reply:
Mike suggested to keep multipartMime batch related class in a namespace back in March as I remember. As I understand, namespace is a logical grouping, shouldn't be solely depends on number of classes as long as it helps keep the project organized.

---
In reply to: [140619605](https://github.com/OData/odata.net/pull/960#discussion_r140619605) [](ancestors = 140619605)

Reply:
Fx guidelines do indicate that number of members in a namespace does matter.

---
In reply to: [140619605](https://github.com/OData/odata.net/pull/960#discussion_r140619605) [](ancestors = 140619605)

Reply:
can you share the Fx guideline so that we can get consensus with Mike?

---
In reply to: [140620805](https://github.com/OData/odata.net/pull/960#discussion_r140620805) [](ancestors = 140620805,140619605)

Reply:
[https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/)

---
In reply to: [140621406](https://github.com/OData/odata.net/pull/960#discussion_r140621406) [](ancestors = 140621406,140620805,140619605)

Reply:
Discussed with mikep, we think it is OK to put these classes in separate namespace.

---
In reply to: [140855510](https://github.com/OData/odata.net/pull/960#discussion_r140855510) [](ancestors = 140855510,140621406,140620805,140619605)